### DISCUSSION TITLE
Best way to re-create this kind of element (in code)?

### DISCUSSION THREAD

**Discussion by mostl33** [mozman/ezdxf#1089] [Q&A]
Hey there,

can anyone recommend a way to re-create this kind of "line" "pattern" with an arbitary height in mm?

Low-res screen grabs:

![image](https://github.com/mozman/ezdxf/assets/73604048/87de6216-092c-4f1a-ac62-2aa5ab2b51cf)

or 

![image](https://github.com/mozman/ezdxf/assets/73604048/3fd53ad3-d063-42e4-b9ef-093f632048ac)

Can "Blocks" with an applied clipping path work? Or do I need shapes?

Basically I am trying to recreate "filled" rectangles (but arbitary height) to visualize layers of insulation (e.g. "Bitumen")

Many thanks (!) in advance.

Best regards,
ml
